sandbox/registration-processor-mz.properties

```diff
@@ -36,6 +36,9 @@ registration.processor.LANDING_ZONE = /mnt/regproc/landing
 #workel pool size to process multiple requests parallely 
 worker.pool.size=10
 
+registration.processor.main-process=NEW,UPDATE,LOST,RES_UPDATE,ACTIVATE,DEACTIVATE
+registration.processor.sub-process=BIOMETRIC_CORRECTION
```

Comment:
Properties placement location is fine, for other please follow the same comments mentioned for these 2 properties in the dmz file

Reply:
This Needs discussion with you and Monobikash

Reply:
Since this requires changes in many stages, it is planned to be done later, so revolved for now